pay-api/tests/unit/api/test_eft_short_names.py

```diff
@@ -431,6 +431,10 @@ def test_search_eft_short_names(session, client, jwt, app):
     assert_short_name(result_dict['items'][0], short_name_2, s2_transaction1)
 
 
+@pytest.mark.skip(reason='This needs to be re-thought, the create cfs invoice job should be handling receipt creation'
```

Comment:
👍 

@rodrigo-barraza, this relates to some of the things we discussed about receipt management now that we are using CFS